tests/script-based-pre/stubbing_compiler_sessions/check_compiler_sessions.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+# Checks that the Kani compiler can encode harnesses with the same set of stubs
+# in one rustc session.
+
+set +e
+
+log_file=output.log
+
+kani stubbing.rs --enable-unstable --enable-stubbing --verbose >& ${log_file}
+
+echo "------- Raw output ---------"
+cat $log_file
+echo "----------------------------"
+
+# We print the reachability analysis results once for each session.
+# Once we unify the stats per Kani compiler run, we should include number of
+# sessions to the result.
```

Comment:
I don't understand this comment. Why is reachability analysis important here? I can see from the code that we grep and print the reachability analysis results, so the comment is a bit redundant. It'd be nice to explain instead why this is important here. Also, is the unification of stats for the Kani compiler being track in an issue? If yes, we should include the link here so we can address it later. Does the issue include the request for the number of sessions?

Reply:
This is important because it is the only way today to reliably get the number of sessions from the compiler without relying on debug comments.

Reply:
Awesome! I'd just add this to the comment.